gh.mk

```diff
@@ -0,0 +1,47 @@
+ifneq ($(PROCURSUS),1)
+$(error Use the main Makefile)
+endif
+
+SUBPROJECTS += gh
+GH_VERSION  := 1.1.0
+DEB_GH_V    ?= $(GH_VERSION)
+
+gh-setup: setup
+	wget -q -nc -P $(BUILD_SOURCE) https://github.com/cli/cli/archive/v$(GH_VERSION).tar.gz
```

Comment:
Please don't download with only version identifier as the tar. Look at brotli.mk for an example on how to do it properly in this case.